sdk/storage/azure-storage-file/azure/storage/file/_generated/models/_models.py

```diff
@@ -609,10 +609,7 @@ class SharePermission(Model):
     }
 
     _attribute_map = {
-        'permission': {'key': 'permission', 'type': 'str', 'xml': {'name': 'permission'}},
-    }
-
-    _xml_map = {
+        'permission': {'key': 'permission', 'type': 'str'},
```

Comment:
we should avoid manual edits to the generated files, right?

Reply:
@zezha-msft - you're correct - this is a temporary workaround for the json/xml feature.

Reply:
It seems we can avoid it once all libraries which depend on msrest upgraded on the next release

Reply:
That's correct. This will be reverted as soon as msrest is updated.